src/frontend/src/containers/pages/dashboard/NewDashboardPage.tsx

```diff
@@ -0,0 +1,20 @@
+import { useBaseUrl } from '@hooks/useBaseUrl';
+import { FC } from 'react';
+
+export const NewDashboardPage: FC = () => {
+  const { businessArea } = useBaseUrl();
+  const dashboardUrl = `${window.location.origin}/api/rest/dashboard/${businessArea}/`;
```

Comment:
what about "global" business area, I'm not sure if this is used but user can have permission to "global" business area

https://dev-hope.unitst.org/global/programs/all/country-dashboard

Reply:
I will build a custom report for the Global business area but as of now, it shows the same for figures/charts as the ones for the normal Business areas.